src/app/components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Input from './Input';
+import type { InputProps } from './Input';
+import type { Story } from '@storybook/react';
+
+export default {
+  title: 'Component/Input',
+  component: Input,
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+  },
+};
+const onChangeFunction = () => {
+  console.log('Hello World');
```

Comment:
You could display whatever value is received here
```suggestion
const onChangeFunction = (value: string) => {
  console.log(value);
```